bdai_ros2_wrappers/test/test_rosbag_recorder.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2023 Boston Dynamics AI Institute Inc.  All rights reserved.
+import os
+import unittest
+
+import rclpy
+from rclpy.node import Node
+from std_msgs.msg import String
+
+from bdai_ros2_wrappers.rosbag_recorder import ROSBagRecorder
+
+BAG_FILE_DIR = os.path.join(os.getenv("BDAI", "/workspaces/bdai"), "recording")
```

Comment:
This is an open-source repo for those outside of bdai, so can you set up the test assuming it is not being run inside the docker

Reply:
Done!